asyncpg/connection.py

```diff
@@ -909,6 +930,25 @@ def _cancel_current_command(self, waiter):
 
         self._loop.create_task(cancel())
 
+    def _notice(self, message):
```

Comment:
Right. Well, you should use `loop.call_soon` anyways. We never want to call callback right away.